src/filters.js

```diff
@@ -119,10 +119,12 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        str = (str === null || str === undefined) ? '' : str;
```

Comment:
I can't trigger CI, you can change commit hash and push it with force. It would trigger Appveyour